### PR TITLE
(DOCSP-18240): Typo fixes, add method names, add code clues in comments

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-18240

### Staged Changes (Requires MongoDB Corp SSO)

- [.NET (Xamarin) Tutorial](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-18240/tutorial/dotnet/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM

File: source/includes/steps-tutorial-dotnet.yaml [hunk 1]
```diff
@@ -176,7 +178,9 @@ content: |
   The ``ProjectPage.xaml.cs`` file is the default screen for this app. In the 
   ``OnAppearing()`` method, we check to see if we have an authenticated user 
   already. If not, we redirect to the Login page. If we do, or when control is 
-  returned from the Login page, we can then instantiate a
+  returned from the Login page, we go to the ``LoadProjects()`` method.
```
Comment (MongoCaleb) [inline source/includes/steps-tutorial-dotnet.yaml:181]: We don't really "go to" a method. Can we find a middle ground, like "we instantiate a SyncConfig object in the LoadProjects method..."?

File: source/includes/steps-tutorial-dotnet.yaml [hunk 2]
```diff
@@ -186,12 +190,14 @@ content: |
...
   Projects in the UI. 
 
-  To implement this functionality, find the ``OnAppearing()`` method in 
-  ``ProjectPage.xaml.cs``. You see the conditional checking to see if we already 
-  have an authenticated user. In the ``else`` block (we have an authenticated user),
-  instantiate the ``userRealm`` object by creating a ``SyncConfiguration`` object 
-  and passing that to the ``Realm.GetInstanceAsync()`` method. When building the 
-  SyncConfiguration object, we set the partition string to ``user=<USER_ID>``:
+  To implement this functionality, find the ``LoadProjects()`` method in 
+  ``ProjectPage.xaml.cs``. We won't reach this method until we have an
```
Comment (MongoCaleb) [inline source/includes/steps-tutorial-dotnet.yaml:194]: Again, this wording of us arriving at a method feels a bit too informal/weird to me. Maybe "this method is not called until we have an authenticated user"?

File: tutorial/dotnet/realm-tutorial-dotnet/LoginPage.xaml.cs [hunk 3]
```diff
@@ -29,7 +29,7 @@ private async AsyncTask DoLogin()
                 var user = await App.RealmApp.LogInAsync(Credentials.EmailPassword(email, password));
                 // :state-end: :state-uncomment-start: start
                 //// TODO: pass the email and password properties to LogInAsync
-                //// user = await ...
+                //// var user = await ...
```
Comment (MongoCaleb) [inline tutorial/dotnet/realm-tutorial-dotnet/LoginPage.xaml.cs:32]: I don't want to check this out to verify: are you sure you need the 'var' here? The user object is defined immediately above on line 29. no?

**Review by MongoCaleb — APPROVED** [review]
A couple of wording questions and one code sanity-check. Thanks!

File: tutorial/dotnet/realm-tutorial-dotnet/LoginPage.xaml.cs (hunk 3 above)
Comment (dacharyc) [reply tutorial/dotnet/realm-tutorial-dotnet/LoginPage.xaml.cs:32]: Maybe I'm reading the Bluehawk markup wrong, but it looks to me like the `var user` is in the final state, but won't be in the start state where this comment is displayed. I haven't done a ton with Bluehawk states, though, so I could be misinterpreting the markup. Is that incorrect?

**Pull request merged**